query-planner-js/src/buildQueryPlan.ts

```diff
@@ -51,6 +53,24 @@ import {
 import { getFieldDef, getResponseName } from './utilities/graphql';
 import { MultiMap } from './utilities/MultiMap';
 import { getFederationMetadataForType, getFederationMetadataForField } from './composedSchema';
+import { DebugLogger } from './utilities/debug';
+
+
+function strIsTrue(str?: string) : boolean {
```

Comment:
This is a small nit, but I usually try to avoid abbreviations in function names.